src/components/popover.vue

```diff
@@ -122,6 +123,7 @@ export default {
 @import '../assets/scss/variables';
 
 .popover {
+  background-color: transparent;
```

Comment:
This was actually helpful for displaying the border radius on the hover card. Without this, the radius is less smooth.